src/io/flutter/sdk/FlutterSearchableOptionContributor.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.sdk;
+
+import com.intellij.ide.ui.search.SearchableOptionContributor;
+import com.intellij.ide.ui.search.SearchableOptionProcessor;
+import io.flutter.FlutterBundle;
+import io.flutter.FlutterConstants;
+import org.jetbrains.annotations.NotNull;
+
+public class FlutterSearchableOptionContributor extends SearchableOptionContributor {
+  @Override
+  public void processOptions(@NotNull SearchableOptionProcessor processor) {
+    add(processor, FlutterBundle.message("settings.try.out.features.still.under.development"));
+    add(processor, FlutterBundle.message("settings.show.preview.area"));
+    add(processor, FlutterBundle.message("settings.experimental.flutter.logging.view"));
+    add(processor, FlutterBundle.message("settings.enable.android.gradle.sync"));
+    // For some reason the word "report" is ignored by the search feature, but the other words work.
+    add(processor, FlutterBundle.message("settings.report.google.analytics"));
+    add(processor, FlutterBundle.message("settings.enable.verbose.logging"));
+    add(processor, FlutterBundle.message("settings.format.code.on.save"));
+    add(processor, FlutterBundle.message("settings.organize.imports.on.save"));
+    add(processor, FlutterBundle.message("settings.flutter.version"));
+    add(processor, FlutterBundle.message("settings.open.inspector.on.launch"));
+    add(processor, FlutterBundle.message("settings.hot.reload.on.save"));
+    add(processor, FlutterBundle.message("settings.disable.tracking.widget.creation"));
+  }
+
+  private static void add(@NotNull SearchableOptionProcessor processor, @NotNull String key) {
+    processor.addOptions(key, null, null, FlutterConstants.FLUTTER_SETTINGS_PAGE_ID,
```

Comment:
@stevemessick @pq Note that if the 3rd argument (`hit`) is not null then the options will also become searchable via Find Action. That's the text that you'll see in Find Action results.

And one more Pro tip (not sure you need it, just FYI): `processor.addOptions()` allows setting aliases. For example, for checkbox called 'foo' you may set alias 'bar' by calling `processor.addOptions("foo bar", null, "You are lucky to find foo via Find Action", "page.id", null, false/true - no idea)` - you'll find 'foo' both by typing 'foo' or 'bar'.